queries/tact/highlights.scm

```diff
@@ -0,0 +1,311 @@
+; NOTE: Neovim's highlight order differs from Tree-sitter's and Helix's,
+; NOTE: as broad captures should be placed first, narrow queries - second.
+; See: https://github.com/nvim-treesitter/nvim-treesitter/blob/master/CONTRIBUTING.md#highlights
```

Comment:
I don't think that's necessary (or wanted) here...

Reply:
Should I remove similar notes from other files as well? I was keeping them for my own reference near the parser, but here they're redundant, I suppose :)

Reply:
Yes, other places as well (and tree-sitter already has changed and Helix will change their precedence to match, so this will be outdated soon).